pkg/api/unversioned/group_version.go

```diff
@@ -46,6 +46,17 @@ func (gr *GroupResource) String() string {
 	return gr.Resource + "." + gr.Group
 }
 
+// ParseGroupResource turns "resource.group" string into a GroupResource struct. It reports error
+// if it cannot parse the string.
```

Comment:
Ok. Then perhaps we should remove "It reports error if it cannot parse the string" from the comment?